packages/components/slider/src/use-slider.ts

```diff
@@ -185,31 +191,37 @@ export function useSlider(originalProps: UseSliderProps) {
   const {direction} = useLocale();
 
   const clampValue = useCallback(
-    (valueToClamp: number) => Math.min(Math.max(valueToClamp, minValue), maxValue),
-    [minValue, maxValue],
+    (valueToClamp: number) => {
+      if (isFixedValue) return minValue;
+
+      return Math.min(Math.max(valueToClamp, minValue), maxValue);
+    },
+    [minValue, maxValue, isFixedValue],
```

Comment:
Undo it. `clampValue` is called from `validatedValue`, where you return `minValue` if `isFixedValue` is `true`. Hence, the change you added won't take any effect.